draft-yasskin-http-origin-signed-responses.md

```diff
@@ -351,21 +350,34 @@ This algorithm also accepts an `allResponseHeaders` flag, which insists that
 there are no non-significant response header fields in the exchange.
 
 1. Let `originalExchange` be the signature's exchange.
-1. Let `exchange` be the significant parts ({{significant-parts}}) of
-   `originalExchange`. If `originalExchange` has no significant parts, then
+1. Let `headers` be the significant headers ({{significant-headers}}) of
+   `originalExchange`. If `originalExchange` has no significant headers, then
    return "invalid".
-1. If `allResponseHeaders` is set and the response headers fields in
+1. Let `payload` be the payload body (Section 3.3 of {{!RFC7230}}) of
+   `originalExchange`. Note that the payload body is the message body with any
+   transfer encodings removed.
+1. If `allResponseHeaders` is set and the response header fields in
    `originalExchange` are a proper superset of the response header fields in
-   `exchange`, then return "invalid".
+   `headers`, then return "invalid".
 1. Let:
    * `signature` be the signature (binary content in the parameterised value's
      "sig" parameter).
+   * `integrity` be the signature's "integrity" parameter.
    * `certUrl` be the signature's "certUrl" parameter, if any.
    * `certSha256` be the signature's "certSha256" parameter, if any.
    * `ed25519Key` be the signature's "ed25519Key" parameter, if any.
    * `date` be the signature's "date" parameter, interpreted as a Unix time.
```

Comment:
[Not for this commit] Should this be a http://www.ietf.org/rfc/rfc2616.txt http-date?

Reply:
It's possible the HTTPWG will tell me to use text dates, but I'd rather avoid the change until then: text dates are longer, take more code to parse and generate, and we can provide the human readability in devtools if we want.